src/middlewared/middlewared/plugins/system_/dmi.py

```diff
@@ -46,7 +49,9 @@ def _parse_dmi(self, lines):
                 continue
 
             sect, val = [i.strip() for i in line.split(':', 1)]
-            if sect == 'Manufacturer':
+            if sect == 'Release Date':
+                SystemService.CACHE['release-date'] = val
```

Comment:
Can we parse it into `date` instance here to avoid doing that in every consumer?